extensions-core/lookups-cached-global/src/main/java/org/apache/druid/server/lookup/namespace/cache/CacheScheduler.java

```diff
@@ -235,10 +235,10 @@ private void tryUpdateCache(String currentVersion) throws Exception
             if (previousCacheState instanceof VersionedCache) {
               ((VersionedCache) previousCacheState).close();
             }
-            log.debug("%s: the cache was successfully updated", this);
+            log.info("%s: the cache was successfully updated", this);
```

Comment:
Will the toString of `this` be large?

Reply:
It would look something like this

```
namespace [JdbcExtractionNamespace{connectorConfig=DbConnectorConfig{createTables=true, connectURI='jdbc:mysql://connectionString', user=userName, passwordProvider=org.apache.druid.metadata.DefaultPasswordProvider, dbcpProperties=null}, table='my_table_name', keyColumn='my_key_column', valueColumn='my_value_column', tsColumn='null', filter='null', pollPeriod=PT0S}] : 
org.apache.druid.server.lookup.namespace.cache.CacheScheduler$EntryImpl@1234abc```

Reply:
Could we reduce that? looks like too verbose. We can log just name, time and size.